tests/expr_and_series/arithmetic_test.py

```diff
@@ -38,6 +38,8 @@ def test_arithmetic_expr(
     constructor: Constructor,
     request: pytest.FixtureRequest,
 ) -> None:
+    if "duckdb" in str(constructor) and attr == "__floordiv__":
```

Comment:
`__floordiv__` [should be implemented](https://github.com/narwhals-dev/narwhals/pull/1725/files#diff-fd9789596cec7305cb278fa984a607015a8f4b1bc664f2da5c36f261c02a42c0R195), or am I looking at the wrong thing?

Reply:
it behaves differently though, so i think we need a separate discussion for how to deal with it, e.g.
```python
In [3]: duckdb.sql('select 1.5 // 2.5')
Out[3]: 
┌──────────────┐
│ (1.5 // 2.5) │
│    double    │
├──────────────┤
│          0.6 │
└──────────────┘

In [4]: 1.5 // 2.5
Out[4]: 0.0
```